development/highlights/storybook.js

```diff
@@ -65,27 +70,22 @@ async function getLocalDependencyList(filename) {
 }
 
 function urlForStoryFile(filename, artifactBase) {
-  const storyId = sanitize(filename);
+  const storyId = getStoryId(filename);
   return `${artifactBase}/storybook/index.html?path=/story/${storyId}`;
 }
 
 /**
- * Remove punctuation and illegal characters from a story ID.
- * See:
- * https://gist.github.com/davidjrice/9d2af51100e41c6c4b4a
- * https://github.com/ComponentDriven/csf/blame/7ac941eee85816a4c567ca85460731acb5360f50/src/index.ts
- *
- * @param {string} string - The string to sanitize.
- * @returns The sanitized string.
+ * @param {fileName} string - The fileName to get the story id.
```

Comment:
Nit: Could we add a description here as well? Ideally all of our new functions would have a description.

We require this with a lint rule in other repos, but it hasn't been enabled in the extension yet due to the huge backlog of descriptions we'd have to add.

```suggestion
 * Get the ID for a story file.
 *
 * @param {fileName} string - The filename of the story file.
```

Reply:
Updated in [17344](https://github.com/MetaMask/metamask-extension/pull/17344)